src/main/java/uk/gov/ida/matchingserviceadapter/configuration/EuropeanConfigurationDefaultsHelper.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ida.matchingserviceadapter.configuration;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.eclipse.jetty.util.TypeUtil.asList;
+
+public class EuropeanConfigurationDefaultsHelper {
+
+    private static final String[] ACCEPTABLE_HUB_CONNECTOR_IDS_PRODUCTION = new String[] {
+        "https://www.signin.service.gov.uk/SAML2/metadata/connector",               // (AWS)
+        "https://connector-node.london.verify.govsvc.uk/SAML2/ConnectorMetadata",   // (GSP)
```

Comment:
```suggestion
        "https://connector-node.london.verify.govsvc.uk/ConnectorMetadata",   // (GSP)
```
This is the correct URL

Reply:
👍